client/src/components/Footer.js

```diff
@@ -10,25 +13,48 @@ function subscribe(e) {
 }
 
 const Footer = () => {
+  const { currentLanguage } = useGlobalContext();
+  const [newsletterPlaceholderText, setNewsletterPlaceholderText] = useState('');
+
+  useEffect(() => {
+    switch (currentLanguage) {
+      case 'ro':
+        setNewsletterPlaceholderText('Introdu adresa de e-mail si apasă ENTER');
+        break;
+
+      case 'hu':
+      default:
+        setNewsletterPlaceholderText('Insert the e-mail address and press ENTER');
+    }
+  }, [currentLanguage]);
+
   return (
     <div className="App-footer">
       <div className="footer-inside">
         <Row type="flex" justify="space-between" gutter={[0, 40]}>
           <Col md={8} xs={24}>
-            <Title level={4}>Link-uri utile</Title>
+            <Title level={4}>
+              <Trans>Useful links</Trans>
+            </Title>
             <Row>
               <a href="/termeni-si-conditii">
-                <Text>Termeni şi condiţii</Text>
+                <Text>
+                  <Trans>Terms &amp; conditions</Trans>
+                </Text>
               </a>
             </Row>
             <Row>
               <a href="/politica-de-confidentialitate">
-                <Text>Politica de confidenţialitate</Text>
+                <Text>
+                  <Trans>Privacy policy</Trans>
+                </Text>
               </a>
             </Row>
             <Row>
               <a href="https://code4.ro/ro/codul-de-conduita">
```

Comment:
Would like to see the links internationalised as well (I'm pretty sure we have versions for `ro`/`en`/`hu` for all links and only the language part should be changed)